fem/pnonlinearform.cpp

```diff
@@ -217,6 +217,28 @@ void ParBlockNonlinearForm::SetEssentialBC(const
    }
 }
 
+double ParBlockNonlinearForm::GetEnergy(const Vector &x) const
+{
+
```

Comment:
Empty line. Shouldn't astyle enforce that?

Reply:
Enforced by hand

Reply:
Unit test added!